### PR TITLE
Minor grammar edits

### PR DESCRIPTION
**Pull request by sfc-gh-mheavin**
### Pre-review checklist
   * [ ] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [ ] I've added new automated tests to verify correctness of my new code.
   * [ ] I've confirmed that my changes are up-to-date with the target branch.
   * [ ] I've described my changes in the release notes.
   * [ ] I've described my changes in the section below.

### Changes description
...

### CODE REVIEW

**Review by sfc-gh-jsikorski — APPROVED** [review]

File: src/snowcli/cli/nativeapp/commands.py [hunk 1]
```diff
@@ -37,13 +37,13 @@
...
-        ..., help="Name of the Native Apps project to be initiated."
+        ..., help="Name of the Native Apps project to initiate."
     ),
     template_repo: str = typer.Option(
         None,
-        help=f"""A git URL to a template repository, which can be a template itself or contain many templates inside it.
-        Example: https://github.com/snowflakedb/native-apps-templates.git for all official Snowflake templates.
-        If using a private Github repo, you may be prompted to enter your Github username and password.
+        help=f"""Specifies the git URL to a template repository, which can be a template itself or contain many templates inside it,
+        such as https://github.com/Snowflake-Labs/native-apps-templates.git for all official Snowflake templates.
```
[the remark this answers is not in the document]
Comment (sfc-gh-mheavin) [reply src/snowcli/cli/nativeapp/commands.py]: Fixed. I was asked to make the changes in the docs after I created this PR.

File: src/snowcli/cli/nativeapp/commands.py [hunk 2]
```diff
@@ -49,9 +49,9 @@ def app_init(
     ),
```
Comment (sfc-gh-jsikorski) [inline src/snowcli/cli/nativeapp/commands.py:49]: The help string for name option was much larger in the main branch version, so while merging, i accepted this version - so please review it again, as it is a new tekst

Comment (sfc-gh-bdufour) [reply src/snowcli/cli/nativeapp/commands.py:49]: LGTM

**Review by sfc-gh-bdufour — DISMISSED** [review]

File: src/snowcli/cli/nativeapp/commands.py [hunk 3]
```diff
@@ -62,7 +62,7 @@ def app_init(
     **options,
 ) -> CommandResult:
     """
-    Initialize a Native Apps project, optionally with a --template-repo and a --template.
+    Initializes a Native Apps project.
```
Comment (sfc-gh-bgoel) [inline src/snowcli/cli/nativeapp/commands.py:65]: This makes sense @sfc-gh-mheavin . His explanation being - we dont list options in the description of other commands, and we should stick to that here as well. I agree with this.

**Review by sfc-gh-bgoel — APPROVED** [review]
LGTM from NADE.

**Review by sfc-gh-bgoel — APPROVED** [review]
LGTM from NADE

**Review by sfc-gh-jsikorski — APPROVED** [review]

**Pull request merged**